Makefile

```
@@ -0,0 +1,9 @@
+.PHONY: install run test clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)